src/requestManager.cpp

```diff
@@ -46,8 +46,11 @@ unsigned int ACCEPTABLE_PING_USEC = 25*1000;
 
 // When should I request an object from someone else (in microseconds)
 unsigned int MIN_TX_REQUEST_RETRY_INTERVAL = DEFAULT_MIN_TX_REQUEST_RETRY_INTERVAL;
+unsigned int txReqRetryInterval = MIN_TX_REQUEST_RETRY_INTERVAL;
 // When should I request a block from someone else (in microseconds)
 unsigned int MIN_BLK_REQUEST_RETRY_INTERVAL = DEFAULT_MIN_BLK_REQUEST_RETRY_INTERVAL;
+unsigned int blkReqRetryInterval = MIN_BLK_REQUEST_RETRY_INTERVAL;
```

Comment:
same as above

Reply:
the reason we do that is apparent if you go to line 467...we have to reset and then recalculate the blkReqRetryInterval each pass through otherwise for nodes that are rate limited we'll end up with  a blkReqRetryInterval that keeps getting larger and larger.  So we have to start with the MIN_BLK_REQUEST_RETRY_INTERVAL which is also a tweak and can change.  It's some logic that we should probably simplify at some point in another PR, but for now I"m just following the way it currently works.

Reply:
fair enough